### PR TITLE
Fixed get_backend selection command

### PR DESCRIPTION
**Pull request by adhishagc**
IBMQ.get_backend('ibmq_16_melbourne') function should be called by the provider object as per the new update.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,8 +31,10 @@ from qiskit import Aer, IBMQ  # import the Aer and IBMQ providers
 from qiskit.providers.aer import noise  # import Aer noise models
 
 # Choose a real device to simulate
-IBMQ.load_accounts()
-device = IBMQ.get_backend('ibmq_16_melbourne')
+token = 'your-ibm-account-token-here'
+IBMQ.save_account(token)
```
Comment (chriseclectic) [inline README.md]: I don't think we need to have the save token example here, just loading a saved account. Can you change to:

```suggestion
provider = IBMQ.load_account()
device = provider.get_backend('ibmq_ourense')
```

**Comment by adhishagc** [comment]
Hi @chriseclectic, I made the change as required.

**Comment by chriseclectic** [comment]
@adhishagc Just one more change: you forgot to remove the `IBMQ.save_account(token)` line.

**Comment by adhishagc** [comment]
@chriseclectic Thank you for pointing it out. Removed the IBMQ.save_account(token) line.

**Comment by adhishagc** [comment]
However
IBMQ.load_accounts()
device = IBMQ.get_backend('ibmq_16_melbourne')

didn't work directly without the initial codelines i had.

**Pull request merged**